package.json

```diff
@@ -56,7 +57,7 @@
     "start": "run-s start:localstack start:service",
     "migrate": "npm run migrate --workspaces --if-present",
     "start:localstack": "AWS_REGION=eu-west-1 run-s localstack:init",
-    "start:service": "cross-env AWS_REGION=eu-west-1 SSM_ENDPOINT=http://localhost:4566 PORT=8080 npm run start -w hrm-service",
+    "start:service": "cross-env AWS_REGION=eu-west-1 SSM_ENDPOINT=http://localhost:4566 PORT=8080 npm run start --w hrm-service",
```

Comment:
Is the extra `-` in `--w` a cross platform thing? It works either way on mac (and I'm guessing will on linux as well). I probably need to update other script entries to use that format if it is required for windows.

Reply:
No I was fiddling around with the scripts because I thought they weren't working until I realised I need a different version of node for Windows to the one in the .nvmrc 🤦 

This change is unintended & almost certainly unnecessary, backing it out